src/site/content/en/blog/logical-property-shorthands/index.md

```diff
@@ -0,0 +1,450 @@
+
+{% Compare 'better', 'Logical shorthand' %}
+```css
+position: absolute;
+inset-block: 10px;
+```
+{% endCompare %}
+</div>
+
+<br>
+
+<div class="w-columns">
+{% Compare 'better', 'Physical longhand' %}
+```css
+position: absolute;
+left: 10px;
+right: 20px;
+```
+{% endCompare %}
+
+{% Compare 'better', 'Logical shorthand' %}
+```css
+position: absolute;
+inset-inline: 10px 20px;
+```
+{% endCompare %}
+</div>
+
+Further reading and a [full list of inset shorthand and longhand](https://developer.mozilla.org/en-US/docs/Web/CSS/inset) 
+is available on MDN.
+
+### Border shorthands
+Border, plus its nested `color`, `style` and `width` properties have all got 
+new logical shorthands as well.
+
+{% Glitch 'border-logical-shorthand' %}
+
+<br>
+
+<div class="w-columns">
+{% Compare 'better', 'Physical longhand' %}
+```css
+border-top-color: hotpink;
+border-bottom-color: hotpink;
+```
+{% endCompare %}
+
+{% Compare 'better', 'Logical shorthand' %}
+```css
+border-block-color: hotpink;
+/* or */
+border-block-color: hotpink hotpink;
+```
+{% endCompare %}
+</div>
+
+<br>
+
+<div class="w-columns">
+{% Compare 'better', 'Physical longhand' %}
+```css
+border-left-style: dashed;
+border-right-style: dashed;
+```
+{% endCompare %}
+
+{% Compare 'better', 'Logical shorthand' %}
+```css
+border-inline-style: dashed;
+/* or */
+border-inline-style: dashed dashed;
+```
+{% endCompare %}
+</div>
+
+<br>
+
+<div class="w-columns">
+{% Compare 'better', 'Physical longhand' %}
+```css
+border-left-width: 1px;
+border-right-width: 1px;
+```
+{% endCompare %}
+
+{% Compare 'better', 'Logical shorthand' %}
+```css
+border-inline-width: 1px;
+/* or */
+border-inline-width: 1px 1px;
+```
+{% endCompare %}
+</div>
+
+Further reading and a [full list of border shorthand and longhand](https://developer.mozilla.org/en-US/docs/Web/CSS/border-block) 
+is available on MDN.
+
+## Logical property `<figure>` example
+Let's put it all together in a small example. Logical properties can layout 
+an image with a caption to handle different writing and document directions.
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/logical-property-figure-demo.mp4">
+  </video>
+</figure> 
+
+Or try it!
+
+{% Glitch 'logical-figure' %}
+
+You don't have to do much to make a card internationally responsive with a 
+`<figure>` and a few logical properties. If you're curious how all this internationally 
+considerate CSS works together, I hope this is a small meaningful introduction.
+
+### Polyfilling and cross-browser support
+The Cascade or build tools are viable options to have old and new browsers alike, 
+properly spaced with updated logical properties. By following a physical property 
+with a logical one, the browser will use the "last" property it found during style 
+resolution. Here's a polyfill by hand:
```

Comment:
Is polyfill the right term here? It's more of a fallback, right? Also, I don't really understand the value of using both? With JavaScript progressive enhancements, I get it. The people with devices that support the advanced features get to use them. But here it seems like we're just duplicating code?

CC @rachelandrew 

(Sorry if you two already discussed this, I checked the Docs comments and didn't see anything)

Reply:
i agree that "polyfill by hand" isn't the right usage, hand written fallback is what it is. good call. 

also agree, there's no value in using both, let's fix where that caused confusion.